script/Arcueid/Day 10/10_00_ARC10_4.txt

```diff
@@ -364,7 +364,7 @@ I wonder how she manages it.
 [sha:eb628e2a1311cded1b85ac21642e7d61cf42f862]{
 -- Page 320, Offset 25067.
 -- 「なに……？　人のことジロジロ見て。
-"...What? It's rude to stare.
+"What? It's rude to stare.
```

Comment:
I kinda like this one tbh, if i remember this va correctly she says it with some apprehension in her voice.